### PR TITLE
Add `NavLink` component

### LINKED ISSUE #235: Add NavLink component

### PR DESCRIPTION
**Pull request by rkillackey**
Resolves #235 

Any thoughts on testing that `activeClassName` is being applied correctly? I tried mocking the router context without much success 😩

### CODE REVIEW

File: src/controls/nav-link.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router'
+
+/**
+ * A component that wraps the `react-router` [`Link`](https://github.com/ReactTraining/react-router/blob/master/packages/react-router-dom/docs/api/Link.md) component, adding an `activeClassName`.
```
Comment (dpikt) [inline src/controls/nav-link.js]: Can you mention here what the default `activeClassName` is?

File: test/controls/nav-link.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { NavLink } from '../../src/'
+
+test('NavLink renders a link tag with props', () => {
+  const wrapper = mount(<NavLink to="/">Home</NavLink>)
+  expect(wrapper.find('a').exists()).toBe(true)
+  expect(wrapper.props().to).toEqual('/')
```
Comment (dpikt) [inline test/controls/nav-link.test.js]: Can you add a check for `activeClassName` being passed down?
You should be able to do `wrapper.find(NavLink)` to get access to the inner component.

**Comment by dpikt** [comment]
@rkillackey re: tasting `activeClassName` I think we might just need to trust react router 😛

**Comment by rkillackey** [comment]
@dpikt Updated!

**Pull request merged**